pkg/minishift/cache/oc_caching_test.go

```diff
@@ -24,11 +24,246 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
 )
 
-var testDir string
-var testOc Oc
+var (
+	openShiftReleaseTagInfo = `{
```

Comment:
If someone else needs it, it is straight forward to extract. There are pro and cons for embedding. The moment another tests needs it lets extract it.